provision/pull-images.sh

```diff
@@ -19,6 +19,7 @@ if [ -z "${NAME_PREFIX}" ]; then
         k8s.gcr.io/coredns:1.6.2 \
         k8s.gcr.io/coredns:1.6.5 \
         k8s.gcr.io/coredns:1.7.0 \
+        k8s.gcr.io/coredns/coredns:v1.8.0 \
```

Comment:
I think we might be able to drop 1.2.6 and 1.3.1 above (cannot comment on the respective lines). They don't seem to be used after cilium/cilium#15502.